packages/amplication-code-gen-types/src/plugin-events-params.ts

```diff
@@ -93,3 +96,28 @@ export interface CreateServerPackageJsonParams extends EventParams {
 export interface CreateServerAppModuleParams extends EventParams {
   modulesFiles: Module[];
 }
+
+export interface CreateEntityResolverParams extends EventParams {
+  templatePath: string;
```

Comment:
instead of sending the template Path, we should send the actual template - see the examples of createEntityController

Reply:
see this relevant issue about another event that should also be fixed
https://github.com/amplication/amplication/issues/4156

Reply:
why ? it can be replaced in the after event function. I think we should stick to our logic when it related to our function and dynamic in the after function (it make more sense)

Reply:
in the after, the template is no longer relevant. 
see the draft for the doc here
https://github.com/amplication/amplication/issues/4097